src/platform/forms-system/src/js/review/ArrayField.jsx

```diff
@@ -173,10 +170,10 @@ class ArrayField extends React.Component {
    * and data is already saved through handleSetData, so we just need to change
    * the edting state
    */
-  handleSave(index) {
+  handleSave(index, fieldName) {
```

Comment:
:thinking: `fieldName` is _right,_ but is there something we can name it that's more _clear_ when we call `handleSave(index, 'foo')`? I'm not sure. Maybe just add a JSDoc comment describing what `fieldName` is used for.

Same question for `handleRemove`.

Reply:
`fieldName` is already defined in the `render` function, so I thought it made more sense to maintain the variable name. I'll add a JSDoc comment to both.